tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+    <https://demo.openwisp.io/admin/config/template/5f279920-60fd-4274-b367-450aa4d30004/change/>`_,
+    **feel free to try it out!**
+
+From the OpenWISP navigation menu, go to ``Configurations`` and then
+``Templates``, from here click on the ``Add template``.
+
+.. image:: ../images/create-template.png
+    :target: ../_images/create-template.png
+
+Fill in name, organization, leave type set to "Generic", backend set to
+"OpenWrt", scroll down to the Configuration section, then click on
+"Advanced mode (raw JSON)".
+
+.. image:: ../images/advanced-mode.png
+    :target: ../_images/advanced-mode.png
+
+Before copying the following NetJSON to the advanced mode editor, you will
+need to update these fields to reflect your configuration:
+
+- ``key`` - RADIUS secret should be same as set in NAS
+- ``server`` - RADIUS server authentication IP
+- ``port`` - RADIUS server authentication port
+- ``acct_server`` - RADIUS accounting server IP
+- ``acct_server_port`` - RADIUS accounting server port
+
+.. code-block:: json
+
+    {
+        "interfaces": [
+            {
+                "wireless": {
+                    "network": [
+                        "lan"
+                    ],
+                    "mode": "access_point",
+                    "radio": "radio0",
+                    "ack_distance": 0,
+                    "rts_threshold": 0,
+                    "frag_threshold": 0,
+                    "ssid": "OpenWISP",
+                    "hidden": false,
+                    "wds": false,
+                    "encryption": {
+                        "protocol": "wpa2_enterprise",
+                        "key": "testing123",
+                        "disabled": false,
+                        "cipher": "auto",
+                        "ieee80211w": "0",
+                        "server": "10.8.0.1",
+                        "port": 1822,
+                        "acct_server": "10.8.0.1",
+                        "acct_server_port": 1823
+                    },
+                    "wmm": true,
+                    "isolate": false,
+                    "ieee80211r": false,
+                    "reassociation_deadline": 1000,
+                    "ft_psk_generate_local": false,
+                    "ft_over_ds": true,
+                    "rsn_preauth": false,
+                    "macfilter": "disable",
+                    "maclist": []
+                },
+                "type": "wireless",
+                "name": "wlan0",
+                "mtu": 1500,
+                "disabled": false,
+                "network": "",
+                "mac": "",
+                "autostart": true,
+                "addresses": []
+            }
+        ],
+      "files": [
+          {
+              "path": "/etc/openwisp/pre-reload-hook",
+              "mode": "0700",
+              "contents": "#!/bin/sh\n\n# Ensure radio0 is enabled \nuci set wireless.radio0.disabled='0'\nuci commit wireless"
+          }
+      ]
```

Comment:
Indentation seems inconsistent: here it's 2 spaces, above is 4. Make it consistent.